spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,81 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:user) { create(:user, organisation: user_organisation) }
+  let(:user_organisation) { create(:organisation) }
+  let(:user_location) { create(:location, organisation: user.organisation) }
+
+  let(:user_2) { create(:user, organisation: user_2_organisation) }
+  let(:user_2_organisation) { create(:organisation) }
+  let(:user_2_location) { create(:location, organisation: user_2.organisation) }
+
+  let(:super_admin) { create(:user, super_admin: true) }
+  let(:super_admin_organisation) { create(:organisation) }
+  let(:super_admin_location) { create(:location, organisation: super_admin.organisation) }
+
+  let(:ip_1) { create(:ip, location_id: user_location.id, address: '1.2.3.4') }
```

Comment:
nit: can these be grouped with their respective collections? (so with `user` and `user_2` respectively)

Reply:
Will do